broker/pkg/coordinator/coordinator.go

```diff
@@ -93,6 +93,7 @@ func (c *Coordinator) AttachToServer(server *GRPCServer) {
 		ConsumerHandshake:    c.consumerHandshake,
 		ConsumerConnected:    c.consumerConnected,
 		ConsumerDisconnected: c.consumerDisconnected,
+		GetConsumer:          c.getConsumer,
```

Comment:
Listeners are used so the Coordinator intercepts gRPC actions, not the other way around.